daemon/mgr/container.go

```diff
@@ -22,7 +22,7 @@ import (
 
 	"github.com/go-openapi/strfmt"
 	"github.com/opencontainers/image-spec/specs-go/v1"
-	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-spec/specs-go"
```

Comment:
why delete the specs-go package's alias name ?  what happens to the code referencing the `specs`?